app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result_list = []
+    for person_data in people:
+        Person.people[person_data["name"]]\
+            = Person(person_data["name"], person_data["age"])
```

Comment:
There is no need to perform this explicit operation since it is done already on line 7 during the class initialization
```suggestion
        result_list.append(Person(person_data["name"], person_data["age"])
```
Besides that, do not use for loop here, perform list comprehension instead